src/tests/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 if(ENABLE_TESTING)
     find_package(Boost 1.39.0 REQUIRED COMPONENTS unit_test_framework)
-    foreach(PROG test_glink test_atom test_fragment test_polarsite test_apolarsite)
-      file(GLOB ${PROG}_SOURCES ${PROG}*.cc)
+    foreach(PROG test_glink test_atom test_fragment test_segment test_segmenttype test_polarsite test_apolarsite)
+      file(GLOB ${PROG}_SOURCES ${PROG}.cc)
```

Comment:
Either remove the glob or rename `test_segment’` to `test_segment_base`!

Globbing without a wildcard is just confusing!